modules/op-release-notes-1-9.adoc

```diff
@@ -296,7 +306,10 @@ Workaround: To mitigate this issue, save the output of `tkn pac resolve` in a te
 // Vincent Demeester	
 
 * Before this update, webhook validation was removed on deletion of old versions of invalid objects. With this update, this issue is resolved.	
-// Vincent Demeester @vdeemester	
+// Vincent Demeester @vdeemester
+
+* Before this update, you could not set some fields of `timeouts`. With this update, this issue is resolved, and you can specify some fields of `timeouts` and let the rest of the fields be default.
```

Comment:
If I were a user, I might have some questions when I read this RN:
- Where or how do I find these fields you're talking about?
- Which fields were previously unsettable and are now settable?
- Are **some** fields still unsettable?  
- Where can I see the default values?

Maybe something more like this:
```suggestion
* Before this update, you could not set some `timeouts` fields in the <name of resource or config file>. This update resolves the issue. Now, you can set the values of the following `timeouts` fields: <field a>, <field b>, and <field c>. If you don't set a value for these fields, they use default values. To learn more, see: link:<URL etc.>
```

Reply:
@vdemeester @rupalibehera - Could you guys help me with some inputs here please?

Reply:
The fix is https://github.com/tektoncd/pipeline/pull/5460 .
> This commit updates PipelineRun validation to allow a user to specify only a tasks timeout or only a finally timeout.
Prior to this commit, if a user wanted to specify no Pipeline timeout (i.e. timeouts.pipeline = 0), they would not be able to specify values for the other timeouts fields.

This is on PipelineRun indeed 👼🏼

Reply:
Thanks for providing feedback folks. I have updated the content. PTAL. Thanks.

Reply:
Outdated. See following comment.